--- Makefile~	2010-06-28 13:00:02.000000000 -0600
+++ Makefile	2010-08-02 13:12:28.000000000 -0600
@@ -134,7 +134,7 @@
 .PHONY: install
 install:
 	@mkdir -p $(DESTDIR)$(PREFIX)/$(LIB)
-	libtool --mode=install $(INSTALL_PROGRAM) $(LIBRARY).la $(DESTDIR)$(PREFIX)/$(LIB)
+	libtool --tag=CC --mode=install $(INSTALL_PROGRAM) $(LIBRARY).la $(DESTDIR)$(PREFIX)/$(LIB)
 	@mkdir -p $(DESTDIR)$(PREFIX)/include/$(PKGNAME)
 	$(INSTALL_DATA) $(EXPANDED_INSTALLED_HEADER_FILES) $(DESTDIR)$(PREFIX)/include/$(PKGNAME)
 	@mkdir -p $(DESTDIR)$(PREFIX)/$(LIB)/pkgconfig
@@ -147,7 +147,7 @@
 	$(INSTALL_DATA) $(VAPI_FILE) $(DESTDIR)$(PREFIX)/share/vala/vapi
 
 uninstall:
-	libtool --mode=uninstall rm -f $(DESTDIR)$(PREFIX)/$(LIB)/$(LIBRARY).la
+	libtool --tag=CC --mode=uninstall rm -f $(DESTDIR)$(PREFIX)/$(LIB)/$(LIBRARY).la
 	rm -rf $(DESTDIR)$(PREFIX)/include/$(PKGNAME)
 	rm -f $(DESTDIR)$(PREFIX)/$(LIB)/pkgconfig/$(PKGNAME).pc
 	rm -f $(DESTDIR)$(PREFIX)/share/vala/vapi/$(PKGNAME).vapi
@@ -163,8 +163,8 @@
 $(EXPANDED_OBJ_FILES): $(BUILD_DIR)/%.o: gexiv2/%.cpp $(EXPANDED_HEADER_FILES) $(CONFIG_IN) Makefile
 	@pkg-config --print-errors --exists '$(EXT_PKG_VERSIONS)'
 	@mkdir -p $(BUILD_DIR)
-	libtool --mode=compile $(CXX) -c $(EXT_PKGS_CFLAGS) $(CFLAGS) -I. -o $@ $<
+	libtool --tag=CC --mode=compile $(CXX) -c $(EXT_PKGS_CFLAGS) $(CFLAGS) -I. -o $@ $<
 
 $(LIBRARY_BIN): $(EXPANDED_OBJ_FILES)
-	libtool --mode=link $(CXX) -rpath $(PREFIX)/$(LIB) $(EXPANDED_LO_FILES) $(EXT_PKGS_LDFLAGS) $(CFLAGS) $(LDFLAGS) -o $(LIBRARY_BIN)
+	libtool --tag=CC --mode=link $(CXX) -rpath $(PREFIX)/$(LIB) $(EXPANDED_LO_FILES) $(EXT_PKGS_LDFLAGS) $(CFLAGS) $(LDFLAGS) -o $(LIBRARY_BIN)
 
